tests/test_integration.py

```diff
@@ -26,6 +26,32 @@ def wait_for_element_by_id(id):
             return self.driver.find_element_by_id(id)
         self.wait_for_element_by_id = wait_for_element_by_id
 
+
+    def wait_for_element_by_css_selector(self, selector):
```

Comment:
We've got these two methods in the base class already, do the versions there not suffice?

https://github.com/plotly/dash/blob/1249ffbd051bfb5fdbe439612cbec7fa8fff5ab5/tests/IntegrationTests.py#L24-L33

Reply:
Oh I see - these were actually added to the base class after this PR was opened. Presumably the versions you added here can just be taken out now.